puppetserver/Dockerfile

```diff
@@ -79,7 +79,7 @@ RUN chmod +x /docker-entrypoint.sh /healthcheck.sh /docker-entrypoint.d/*.sh &&
     apt-get update && \
     apt-get install -y --no-install-recommends $PACKAGES $BUILD_PKGS && \
     gem install --no-doc r10k -v $R10K_VERSION && \
-    gem install --no-doc rugged -v $RUGGED_VERSION && \
+    gem install --no-doc rugged -v $RUGGED_VERSION -- --with-ssh && \
```

Comment:
are these double `--` a typo? or what does it do?

Reply:
That is what i found somewhere and it does seem to work. Mind you, I'm probably not going to use rugged as it caused other problems. But having it in there without ssh support seems like it's missing something.

Reply:
okay. thx for the explanation :)